lib/cloud/aws/identity.go

```diff
@@ -23,6 +23,8 @@ import (
 	"fmt"
 	"strings"
 
+	awsV2 "github.com/aws/aws-sdk-go-v2/aws"
```

Comment:
```suggestion
	awsv2 "github.com/aws/aws-sdk-go-v2/aws"
```
Try to keep package names (and their aliases) all lowercase